components/tools/OmeroPy/src/omero/testlib/__init__.py

```diff
@@ -538,7 +538,7 @@ def new_user(cls, group=None, perms=None,
         :system: If user is to be a system group member
         :privileges: If system group member is to have privileges
                      privileges=None gives all privileges (full admin)
-                     privileges=[] gives no privileges
+                     privileges=() gives no privileges
```

Comment:
Yes, that is how it should be, consistent. I forgot about this one, thank you @joshmoore